runtime-decompiler/src/main/java/org/jrd/frontend/ConfigureFrame/ConfigureView.java

```diff
@@ -40,6 +45,10 @@
             this.label = new JLabel(label);
             this.browseButton = new JButton(ButtonLabel);
 
+            chooser = new JFileChooser();
+            File dir = Paths.get("./decompiler_agent/target").normalize().toAbsolutePath().toFile();
```

Comment:
Altough it is muchbetter then it was, it is starting from cwd, where it shouldbe starting from.... jar dir? binary dir?  In developers run? In static run?

Idea - run.sh can set java property )(-jrd.launcher.location)  and then yo can try this proeprty first, or as first fallback the jars location. 
and this is bad for istribution build... Distributio build cna have this hard coded... in sh? IN jar? Tricky thing those defaults....

Reply:
Awww should I remove this commit then for now?

Reply:
not sure. It is mch better tehn it was for developer usecase, and moreover unchanged , but missleading, for portable or distribution builds.

Maybe it can be better to incorporate it to your current work  with new macro and property? (eg use this, but if property exists.. do something else.)